packages/roosterjs-editor-core/lib/corePlugins/CopyPlugin.ts

```diff
@@ -22,6 +23,7 @@ export default class CopyPlugin implements EditorPlugin {
     public initialize(editor: Editor) {
         this.editor = editor;
         this.copyDisposer = editor.addDomEventHandler('copy', this.onCopy);
+        this.cutDisposer = editor.addDomEventHandler('cut', this.onCopy);

```

Comment:
editor.addDomEventHandler() supports adding multiple events:
editor.addDomEventHandler({
copy: this.onCopy,
cut: this.onCut,
});